src/includes/class_reservationPermissions.php

```diff
@@ -53,16 +53,23 @@ public function getBuildings($id = null){
     }
   }
 
-  public function permissionsSet($id = null){
-      // checks to see if the building ID passed is in the reservePermissions table
-      // if it finds it it returns true
+  // permissionsSet
+  // ==================================================
+  // Name : Tracy McCormick
```

Comment:
If you are going to document methods, you should use the php doc block, which is what is the php standard, and what the rest of EngineAPI uses: https://www.phpdoc.org/docs/latest/guides/docblocks.html